neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -73,7 +74,13 @@ public static ContractManifest CreateDefault(UInt160 hash)
                     Hash = hash,
                     EntryPoint = ContractMethodDescriptor.DefaultEntryPoint,
                     Events = new ContractEventDescriptor[0],
-                    Methods = new ContractMethodDescriptor[0]
+                    Methods = new ContractMethodDescriptor[0],
+                    ScriptHeader = new ScriptHeader()
+                    {
+                        Compiler = "",
+                        Engine = ScriptHeader.ScriptEngine.NeoVM,
```

Comment:
We add header for `.avm` files not to allow NEO to support other virtual machines, but to allow other blockchains to support NeoVM. So adding a header to the manifest is meaningless.

Reply:
Only for a contract script, or in all scripts?

Reply:
In avm file, I think. When deployed, the header is removed.

Reply:
Raw encoding 4 bytes prefix on avm? I think it's good too, a good start for "nvm3" format.